adm_program/system/classes/TablePhotos.php

```diff
@@ -220,7 +220,40 @@ public function isVisible()
             return false;
         }
 
-        return true;
+        if($gCurrentUser->isAdministrator()) {
```

Comment:
Why have an administrator the right to view the album?
Since now an administrator have no special rights in the photo album. We should use the `$gCurrentUser->editPhotoRight()` instead.

Reply:
line removed